parasite/scripts/production/core-creation/gff_utils.py

```diff
@@ -22,15 +22,13 @@
 
 # Removes provided prefixes from all columns in a .gff3 dataframe.
 def remove_prefixes_from_column_values(gff_df, prefixes):
-    for prefix in prefixes:
-        # Look only at beginning of line, escape any special characters.
-        prefix = "^"+re.escape(prefix)
-        for col in gff_df:
-            # Avoid using re.sub on NaN values as this turns them into str
-            nan_mask = gff_df.loc[:,col].isna()
-            gff_df.loc[~nan_mask, col] = [re.sub(prefix, "", str(x)) for x in gff_df.loc[~nan_mask, col]]
+    gff_df = gff_df.replace(prefixes, "", regex=True)

```

Comment:
Does this still check that the prefixes being replaced are at the beginning of the value?